### DISCUSSION TITLE
Suggestion: Enable Web Server by Default

### DISCUSSION THREAD

**Discussion by AngryMooses** [gtxaspec/wz_mini_hacks#505] [General]
I just got this setup and working, thanks everyone for the work you've put in!

Suggestion to make this hack easier to use for others would be to enable the web server by default, then point people to the web UI to make any config changes that they'd like.  

Personally I wasted a ton of cycles trying to get SSH working, as the config instructions imply that is the primary way of connecting.  Turns out I didn't need SSH at all, just needed to make edits to the config file by popping the SD card in and out several times.  Once I got the web server running I realized I didn't even need to do that, the web server gave me access to 99% of the things I needed.

Would love to see that as the default.

**Comment by Bob-AOL** [comment]
Never happen and shouldn't in its current state. You can drive a bus through the security holes in the web server..., as in it is 100% wide open to anyone on the network knowing the IP address. Just type it into a browser and you're there. 

Login authentication would be a good start if anyone would take the time to develop it. In the meantime, SSH is the only secure way to manage a camera online. It is simple to set up and use right from a Windows command prompt. No mysterious secret geek knowledge required. There's a fresh thread demystifying SSH that you may already have seen. If not, it may be worth your while to at least check it out. I don't remember the number at the moment, but it may have even been the one previous to this one.

The other option is obviously to pull the SD card and edit offline, but that has its limitations depending on one's objectives and skill level.
As mentioned in several threads, SSH is not a pre-requisite. It just makes sense if you're constantly logging in and out of your cameras to do so in the most secure way possible.

But the good news is, you're up and running. Congrats.
